addrmgr/knownaddress.go

```diff
@@ -69,19 +70,18 @@ func (ka *KnownAddress) chance() float64 {
 		lastAttempt = 0
 	}
 
-	c := 1.0
+	// Apply a lower limit to the value returned.
+	const minChance = 0.01
```

Comment:
Only thing I'm uncertain about is this const value.  I haven't tested this on mainnet, but I don't see how this limit could be too high.

Reply:
I don't see any issue with it considering the total number of peers.  If there were several million nodes it would probably be too high, but that isn't the case.